tests/pipelines/test_pipelines_common.py

```diff
@@ -423,6 +425,33 @@ def test_unbatch_attentions_hidden_states(self):
         self.assertEqual(len(outputs), 20)
 
 
+@require_torch_and_tf
```

Comment:
We cannot allow this, as the tests are run independantly, no test will ever by ran with the two frameworks !


You tests are good, just:
- Remove `parameterized` use two different tests each time, it's much easier  to read and parse (and execute separately).
- Remove `require_torch_and_tf`.

Reply:
@Narsil Many thanks for the review and the suggestion. Updated the PR.